### DISCUSSION TITLE
lossless viruses

### DISCUSSION THREAD

**Discussion by radica17** [mifi/lossless-cut#687] [Support]
im pretty sure this program gave me a virus i have 17 dectectios (pups) wondering if anyone got this?

**Comment by Shakil-Shahadat** [comment]
How did you scan the app? I checked via virus total and got [zero](https://www.virustotal.com/gui/file/f9c38ed2dc0af9be228fe943b9ed3112182577937f53d9fa9d65135636d5e79a/detection) detection.

And virus and pups are completely different thing.

**Reply by mifi** [reply]
Like @Shakil-Shahadat  said I cannot comment on this because I don't know where you downloaded the software and I don't know which application you're using to scan. please provide more info

**Reply by radica17** [reply]
i downloaded from your github no where else it was pups installed in the chrome folder the pup was a mindspark pup

**Reply by radica17** [reply]
> How did you scan the app? I checked via virus total and got [zero](https://www.virustotal.com/gui/file/f9c38ed2dc0af9be228fe943b9ed3112182577937f53d9fa9d65135636d5e79a/detection) detection.
> 
> And virus and pups are completely different thing.

i scanned with malwarebytes when i installed the program

**Reply by Shakil-Shahadat** [reply]
Where did you get the pup version of the app?
